arviz/tests/helpers.py

```diff
@@ -201,9 +205,13 @@ def load_cached_models(draws, chains):
         )
         path = os.path.join(data_directory, fname)
         if not os.path.exists(path):
+
             with open(path, "wb") as buff:
+                _log.info("Generating and caching %s" % fname)
```

Comment:
should use `_log.info("Generating and caching %s", fname)`.  formatting will not occur until the logging happens, and services like sentry that aggregate logs can use the format string to group them together, even if the `fname` is different.

Reply:
Ah shoot, missed the comma. Thanks for the catch